### PR TITLE
DS-205 Replace Popper with FloatingUI in Dropdown component

### PR DESCRIPTION
**Pull request by MichalPaszowski**
Popper is dead - long live Floating UI.
No design changes.

### CODE REVIEW

File: packages/react-components/src/components/Popover/Popover.stories.css [hunk 1]
```diff
@@ -21,3 +21,10 @@
   line-height: 20px;
   margin-left: 5px;
 }
+
+.lc-testArea {
```
Comment (sgraczyk) [inline packages/react-components/src/components/Popover/Popover.stories.css:25]: What is `testArea`?

Comment (marcinsawicki) [reply packages/react-components/src/components/Popover/Popover.stories.css:25]: And why `lc-` prefix?

**Review by marcinsawicki — APPROVED** [review]

File: packages/react-components/src/components/Popover/Popover.stories.css (hunk 1 above)
Comment (MichalPaszowski) [reply packages/react-components/src/components/Popover/Popover.stories.css:25]: I know it is a stupid name but could not come up with something better. Basically, it's a div where the component is tested :P

Comment (MichalPaszowski) [reply packages/react-components/src/components/Popover/Popover.stories.css]: fixed

File: packages/react-components/src/components/Popover/Popover.spec.tsx [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+      };
+    }),
+    autoUpdate: vi.fn(),
+    flip: vi.fn(),
+    offset: vi.fn(),
+    x: 0,
+  };
+});
 
 const PopperContent = () => {
```
Comment (sgraczyk) [inline packages/react-components/src/components/Popover/Popover.spec.tsx:26]: Still using `Popper` wording. Try to avoid library naming at all.

**Review by sgraczyk — APPROVED** [review]

**Pull request merged**